src/init.cpp

```diff
@@ -1659,6 +1654,11 @@ bool AppInitMain(boost::thread_group& threadGroup, CScheduler& scheduler)
     connOptions.nMaxOutboundTimeframe = nMaxOutboundTimeframe;
     connOptions.nMaxOutboundLimit = nMaxOutboundLimit;
 
+    if (gArgs.IsArgSet("-seednode")) {
+        BOOST_FOREACH(const std::string& strDest, gArgs.GetArgs("-seednode"))
```

Comment:
Even though your only moving this, if we're going to touch these lines, can you remove Boost.